### PR TITLE
Added docstrings to clustering tests

### PR DESCRIPTION
**Pull request by arovir01**
This PR improves the documentation of the clustering tests by adding docstrings that explain what each test is supposed to verify.

### CODE REVIEW

**Comment by arovir01** [comment]
@akarmi Please review.

File: tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py [hunk 1]
```diff
@@ -86,19 +108,31 @@ def testCannotBeInitializedWithFloatNumberOfClusters(self):
...
+    clusters.
+    """
     with self.assertRaises(ValueError):
       cluster_wrapper.ClusterWeights(layers.Dense(10),
                                      number_of_clusters=number_of_clusters,
                                      cluster_centroids_init='linear')
 
   def testCanBeInitializedWithAlreadyClusterableLayer(self):
+    """
+    Verifies that ClusterWeights can be initialized with an custom clusterable
```
Comment (akarmi) [inline tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py]: nit: "a custom clusterable layer"

File: tensorflow_model_optimization/python/core/clustering/keras/clustering_centroids_test.py [hunk 2]
```diff
@@ -95,6 +114,10 @@ def testLinearSolverSolveForX(self, x1, y1, x2, y2, x, y):
       (7, 12, 17, 22, 3, 8),
   )
   def testLinearSolverSolveForY(self, x1, y1, x2, y2, x, y):
+    """
+    Verifies that TFLinearEquationSolver solves the given equations correctly
+    for X.
```
Comment (akarmi) [inline tensorflow_model_optimization/python/core/clustering/keras/clustering_centroids_test.py]: "for Y"

**Comment by akarmi** [comment]
@alanchiao, please merge this.

**Comment by alanchiao** [comment]
FYI, @akarmi. We now have a rotation for this who will be checking frequently. Hopefully we can eventually make the rotation public in some manner. 

@liufengdb, @daverim

**Comment by akarmi** [comment]
> 
> 
> FYI, @akarmi. We now have a rotation for this who will be checking frequently. Hopefully we can eventually make the rotation public in some manner.
> 
> @liufengdb, @daverim

Fantastic. Welcome @liufengdb, @daverim.

**Pull request merged**